kaznet/apps/main/serializers/submissions.py

```diff
@@ -13,15 +14,34 @@
                                           LABEL_LOCATION_ID, LABEL_TASK_ID)
 from kaznet.apps.main.models import Submission
 from kaznet.apps.main.serializers.base import GenericForeignKeySerializer
-from kaznet.apps.main.serializers.bounty import SerializableAmountField
 
 
+class SerializableDateField(serializers.Field):
+    """
+    Custom Field submission_time
+    """
+
+    def to_representation(self, value):
+        """
+        Custom to submission_date field
+        """
+        return value.astimezone(pytz.timezone('Africa/Nairobi')).replace(
+            microsecond=0).isoformat()
+
+    def to_internal_value(self, data):
+        """
+        custom internal value for submission_date field
+        """
+        return data
+
 # pylint: disable=too-many-ancestors
+
+
 class KaznetSubmissionSerializer(GenericForeignKeySerializer):
     """
     Main Submission serializer class
     """
-    amount = SerializableAmountField(read_only=True)
+    submission_time = SerializableDateField()
```

Comment:
Perhaps a good test that this method works as expected is to compare a submission object from the DB with the same object passed through the serializer and check that the only difference between the two is that the one from the serializer has no microseconds.